runtime/oti/j9.h

```diff
@@ -391,6 +391,32 @@ typedef struct {
 #define J9_IS_31BIT_INTEROP_TARGET(handle) J9_ARE_ALL_BITS_SET((UDATA)(handle), OMRPORT_SL_ZOS_31BIT_TARGET_HIGHTAG)
 #endif /* defined(J9VM_ZOS_3164_INTEROPERABILITY) */
 
+/* Check if a JVMTI event should be dispatched by checking the flag set by
+ * notifyJvmtiHideFrames() and the presence of the JvmtiMountTransition
+ * annotation.
+ *
+ * The thread should be checked before dispatching the following events:
+ * Exception
+ * Exception Catch
+ * Field Access
+ * Field Modification
+ * Frame Pop
+ * Method Entry
+ * Method Exit
+ *
+ * Additionally, the method should be checked before dispatching the following events:
+ * Frame Pop
+ * Method Entry
+ * Method Exit
+ */
+#if JAVA_SPEC_VERSION >= 20
+#define SHOULD_DISPATCH_HOOK(currentThread, method) \
+    (J9_ARE_NO_BITS_SET((currentThread)->privateFlags, J9_PRIVATE_FLAGS_VIRTUAL_THREAD_HIDDEN_FRAMES) \
+    && (NULL == (method) ? TRUE : J9_ARE_NO_BITS_SET(getExtendedModifiersDataFromROMMethod(J9_ROM_METHOD_FROM_RAM_METHOD((method))), CFR_METHOD_EXT_JVMTIMOUNTTRANSITION_ANNOTATION)))
```

Comment:
Missing parenthese around the `==` subexpression; redundant parentheses for the second use of `method` here.